proposals/vector_reproject.json

```
@@ -44,7 +44,7 @@
         }
     ],
     "returns": {
-        "description": "A vector data cube with geometries converted into the new projection. The reference system of the vector dimension changes, all other dimensions and properties remain unchanged.",
+        "description": "A vector data cube with geometries converted into the new projection. The reference system of the vector dimensions changes all other dimensions and properties remain unchanged.",
```

Why did you remove the comma here? I think it's correct an makes the sentence much easier to read/understand.

---

Not sure either, I'll add it again.

```suggestion
        "description": "A vector data cube with geometries converted into the new projection. The reference system of the vector dimensions changes, all other dimensions and properties remain unchanged.",
```

---

I'm wondering, whether having a dimension parameter as in filter_temporal makes sense?!

---

@m-mohr I guess that you commented in the wrong PR?

---

@clausmichele No, it's the right PR. In theory there could be two geometry dimensions, right?

---

But are you talking about the `filter_temporal` process or `vector_reproject`? In the comment of 2 hours ago you mentioned `filter_temporal`.

---

Yes, filter_temporal also has a dimension parameter and I mentioned that as an example for how it could be done in this process as well.

---

Ok sorry, I misunderstood.
Anyway I agree, probably it could be good to have a dimension parameter with the default set to `geometries`.

---

Yes, but defaults to `geometry` ;-)

---

Added the parameter and added the comma.